host_agent.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+__author__ = "Nash Kaminski"
+__copyright__ = "Copyright 2021, Nash Kaminski"
```

Comment:
Please remove this. Feel free to leave the author in, but we can't accept copyrighted code.

Reply:
__copyright__ line / variable removed.